api/Document.json

```diff
@@ -4911,7 +4911,7 @@
             ],
             "ie": {
               "version_added": "11",
-              "alternative_name": "MSFullscreenChange"
+              "alternative_name": "msfullscreenchange"
```

Comment:
I believe that this and the "MSFullscreenError" change was mistaken, and have sent https://github.com/mdn/browser-compat-data/pull/6684 to change them back.

https://jsbin.com/wizuwejocu/1/edit?html,js,output tested the event handler attributes, but not the event types. These are almost always the same apart from the "on" prefix, but when the event type isn't all lowercase, the event handler attribute usually still is. At least I don't know of any exceptions to that.

Reply:
Ah sorry for misreading. Thank you for getting this fixed.